frontend/components/markup-modal/copied-content-indicator.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import posed from 'react-pose'
+import styled from 'styled-components'
+
+const ContentAnchor = styled.span`
+  position: relative;
+  display: inline-block;
+`
+
+// 100vw allows providing styled content which is wider than its container.
+const ContentContainer = styled.span`
+  width: 100vw;
+
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
Hehe. I'm still learning layout. I fiddled with it for a _long_ time and I'd love to find a better way. 😝 

This is what it looks like now. It glides up, centered over the button as intended. Normally it would then disappear but I turned that off for the screenshot.

<img width="1096" alt="screen shot 2019-01-08 at 12 58 15 pm" src="https://user-images.githubusercontent.com/1487036/50849457-2111da00-1345-11e9-9fd6-a376374eba35.png">


If I remove `left` and `transform` and set `margin: 0  auto` it's way off to the right. It might be centered over the right bound of the parent `position: relative` element; not sure.

<img width="1111" alt="screen shot 2019-01-08 at 12 56 30 pm" src="https://user-images.githubusercontent.com/1487036/50849427-00498480-1345-11e9-953a-044cb3da2817.png">

Btw this is all live in the review app. Click one of the examples with colons, fill in the boxes, and click the button, and you can see what the animation is supposed to do.

Reply:
From a design / UX perspective, I think it would be nicer if didn't have the actual text was copied but just a simple 'copied' as the content of the indicator. I've seen that on other sites used a lot and think that would be a better solution since no one would be able to read the text in that short period of time. On the other hand, it would be nice to see the text before it gets copied to the clipboard.

Reply:
I'm fine with "copied."

We could experiment with showing the actual text. Seeing unfinished markup before you start filling the form looks like a mistake and I think is confusing. We could show an empty rectangle with a message "your markup will appear here" or "some parameters missing," similarly to the way the preview badge is handled.

During editing, I think the changing text will be distracting for some and comforting for others. It's about adding something new, so let's revisit it. If we decide on a design process in #1899 that will put us on good ground for the future work.

Reply:
I agree. I'll write it down and we leave like this here.

Reply:
I updated the indicator to show **Copied**.